### DISCUSSION TITLE
How to make an existing Comment private?

### DISCUSSION THREAD

**Discussion by boschi** [cloudbees/zendesk-java-client#463] [Q&A] [answered]
I need to change a few comments in an existing ticket from public to private.

Zendesk API has this feature here:  https://developer.zendesk.com/api-reference/ticketing/tickets/ticket_comments/#make-comment-private

Is this call implemented in the java-client?

**Comment by aheritier** [comment, marked as answer]
Hi @boschi,

No it's not yet supported AFAICS.

I created a RFE if someone wants to implement it

**Reply by aheritier** [reply]
https://github.com/cloudbees-oss/zendesk-java-client/issues/467
